openid-connect-4-credential-issuance-1_0.md

```diff
@@ -287,39 +295,119 @@ experience, it seems to be the better way to leave implementers the choice. If t
 implementation experience, will gravitate towards one or the other approach, the draft could
 be simplified by removing one of the options. 
 
-# Flow 1: Simple Issuance Flow Overview (without submission of input VCs)
+# Endpoints
+
+This specification defines the set up phase and the following Endpoints. 
+
+* Nonce Endpoint
+* Pushed Authorization Endpoint
+* Authorization Endpoint
+* Token Endpoint
+* Credential Endpoint
+
+## Set up Phase
+
+This phase is OPTIONAL. It is used by the Client prior to initiating the transaction with the Issuer to discover which credentials the Issuer is capabile of issuing and if there are any pre-requisites when requesting the issuance of those credentials. 
+
+After completing the set-up, the Client will discover one or more credential manifests as defined in [@DIF.CredentialManifest]. Credential Manifest contains information about which type of VCs the Issuer can issue, and, optionally, what kind of input the Issuer requires from the Client in the request to issue that credential. 
+
+This allows for a static approach of presenting input credentials to the Issuer as defined in section 6.
+
+### Obtaining the Credential Manifest using OpenID Discovery
```

Comment:
I suggest to elevate this section by one level and designate it as "Server Metadata"